yumi_description/launch/yumi.rviz

```diff
@@ -1,236 +0,0 @@
-Panels:
```

Comment:
They were more or less the same thing as this [rviz file](https://github.com/ethz-asl/yumi/blob/master/yumi_moveit_config/launch/moveit.rviz  "YuMi RViz file") and this [launch file](https://github.com/ethz-asl/yumi/blob/master/yumi_moveit_config/launch/moveit_planning_execution.launch "YuMi Launch file"). The RViz file that was deleted wan't updated to include the camera assembly. Should I put them back or are the linked ones good enough?